src/openrct2/platform/Platform.Posix.cpp

```diff
@@ -31,9 +31,7 @@
 #    include <sys/time.h>
 
 // The name of the mutex used to prevent multiple instances of the game from running
-static constexpr u8string_view SINGLE_INSTANCE_MUTEX_NAME = u8"openrct2.lock";
-
-static utf8 _userDataDirectoryPath[MAX_PATH] = { 0 };
+static constexpr const utf8* SINGLE_INSTANCE_MUTEX_NAME = u8"openrct2.lock";
```

Comment:
should stay `u8string_view`

Reply:
`open` requires a null terminated string, to until we have a `u8string_zview` that guarantees null termination, it can't.